etterlattemaler/src/main/kotlin/no/nav/pensjon/etterlatte/maler/omstillingsstoenad/etteroppgjoer/EtteroppgjoerForhaandsvarsel.kt

```diff
@@ -50,14 +50,15 @@ data class EtteroppgjoerForhaandsvarselBrevDTO(
 
 enum class EtteroppgjoerResultatType{
     TILBAKEKREVING,
-    ETTEROPPGJOER
+    ETTERBETALING,
+    IKKE_ETTEROPPGJOER
 }
 
 data class EtteroppgjoerForhaandsvarselDTO(
     val bosattUtland: Boolean,
     val norskInntekt: Boolean,
     val etteroppgjoersAar: Int,
-    val rettsgebyrBeloep: Kroner,
```

Comment:
Hvorfor bytte fra `Kroner` til `Int`?

Reply:
Hadde problem med det vi sendte, men endrer tilbake til Kroner for rett formatering